040-SDK/030-Go/010-overview.mdx

```diff
@@ -22,7 +22,9 @@ go get github.com/xataio/xata-go@latest
 
 ## Configuration
 
-The client needs a `XATA_API_KEY` environment variables for authentication and the `.xatarc` file for the database URL. Refer to [our installation guide](/docs/getting-started/installation#xatarc) to initialize your project with the `.xatarc` file.
+The client expects a `XATA_API_KEY` environment variables for [authentication](/docs/rest-api/authentication). To set the workspace, region and branch you have two options:
```

Comment:
The client requires the `XATA_API_KEY` environment variable for [authentication](/docs/rest-api/authentication). You can configure your workspace, region, and branch in two ways:

### Using Environment Variables

To configure the environment, begin by setting the `XATA_WORKSPACE_ID`. Use the following command in the terminal:

```sh
export XATA_WORKSPACE_ID="your_workspace_id"
```

- `XATA_WORKSPACE_ID`: This is variable to mandatory.
- `XATA_BRANCH`: This is optional. If unspecified, `XATA_BRANCH` defaults to `main`.  
- `XATA_REGION`: This is optional. Explore [available regions](/docs/getting-started/available-regions) for additional options to `us-east-1`. If unspecified, `XATA_REGION` defaults to `us-east-1`
   
After setting the environment variables in your terminal, your Go application can access them using the [os package](https://pkg.go.dev/os), typically with `os.Getenv`. For example:

```go
workspaceID := os.Getenv("XATA_WORKSPACE_ID")
branch := os.Getenv("XATA_BRANCH")
region := os.Getenv("XATA_REGION")
```

### Using the `.xatarc` file

Alternatively, you can configure settings using the `.xatarc` file. To initialize your project with the `.xatarc` file, refer to the [installation guide](/docs/getting-started/installation#xatarc).